packages/manager/modules/product-offers/src/product-offers.component.js

```diff
@@ -70,6 +70,12 @@ export default {
      */
     workflowOptions: '<',
 
+    /**
+     * Getter for workflow configuration
+     * @type {function}
+     */
+    getWorkflowConfiguration: '&?',
```

Comment:
Hmm why having this binding at this level? Order workflow is supposed to have its own `getConfiguration`method.
If you look here: https://github.com/ovh/manager/blob/master/packages/manager/modules/product-offers/src/workflows/product-offers-order-workflow.class.js#L30

Reply:
The issue is we cannot bind to the current controller. If I set `onGetConfiguration: $ctrl.getConfiguration.bind($ctrl)` (to be able to use `this` inside the `getConfiguration` function) it triggers `digest` errors

Reply:
I was having issue with binding `this` in the `getConfiguration` but now I've thought about it with your comment I might have found a solution (I'll update if this suit the needs)

Reply:
Ok yes that's what I had in mind :)